src/common/accountParser.ts

```diff
@@ -14,12 +15,12 @@ export function pubkeyField(property?: string): PubkeyField {
 }
 
 /**
- * TODO:
+ * TODO: fix <any> type
  * @export
  * @class PubkeyField
  * @extends {BL.Layout}
  */
-export class PubkeyField extends BL.Layout {
+export class PubkeyField extends BL.Layout<any> {
```

Comment:
This should be PublicKey no?
I'm not sure this produces any type errors, you'll have to check
```suggestion
export class PubkeyField extends BL.Layout<PublicKey> {
```